src/panels/lovelace/cards/hui-markdown-card.ts

```diff
@@ -80,7 +80,8 @@ export class HuiMarkdownCard extends LitElement implements LovelaceCard {
       <ha-card .header="${this._config.title}">
         <ha-markdown
           breaks
-          class="markdown ${classMap({
+          class="${classMap({
+            markdown: true,
```

Comment:
revert this change

Reply:
Why?

Reply:
Because there is no need to add it to a `classMap` if you want the class to be there always.

Reply:
I thought it looked cleaner using it for everything when it's in use 🤷‍♂️ 
But you are the boss, so revert it is! 🎉

Reply:
Might be cleaner, but it is definitely more expensive.